styles/Canonical/001-English-words-spelling-suggestions.yml

```diff
@@ -94,7 +91,6 @@ swap:
   'urbanization': urbanisation
   'virtualize': virtualise
   'virtualization': virtualisation
-  'whiskey': whisky
```

Comment:
! Whiskey is the proper way to spell it! 
Both are valid in UK English actually and are technically different drinks (whiskey is usually triple distilled and blended).

Reply:
This is also not the most relevant word for documentation I reckon ... :smile: